packages/core/src/routes/interaction/verifications/mfa-verification.ts

```diff
@@ -114,7 +122,7 @@ const validateMandatoryBindMfaForSignIn = async (
   const { mfaVerifications, customData } = await tenant.queries.users.findUserById(accountId);
 
   // If the user has linked MFA before
```

Comment:
should update comments
`If the user has linked currently available MFA before`?